README.md

```diff
@@ -1,22 +1,69 @@
+
+### Prereqs.
+* You have the Kinect SDK installed and `k4viewer` works for you.
+* You have Python3 development libraries installed. (`apt-get install python3-dev`)
+* You have an Azure Kinect device
+
+### Build/Install
+```sh
+git clone https://github.com/brendandburns/py-k4a.git
+cd py-k4a
+python3 setup.py build
+sudo python3 setup.py install
+python3 example.py
+```
+
+If all goes well, this will leave a file named `test.jpg` in your directory with a single frame capture
+from your device.
+
+### TODO
+Add support for Audio and IMU. Fix bugs. Accept PRs.
+
+## Instructions for Window Install
+This may not be the most elegant or minimal solution, but it worked for me on two different Windows 10 machines.
+
+### Prereqs.
+* You have the Kinect SDK installed and `k4viewer` works for you.
+	* Find (corresponding on your install) following directories/files:
+		* `C:\Program Files\Azure Kinect SDK v1.2.0\sdk\include\k4a`
+		* `C:\Program Files\Azure Kinect SDK v1.2.0\sdk\windows-desktop\x86\release\lib\k4a.lib`
+		* `C:\Program Files\Azure Kinect SDK v1.2.0\sdk\windows-desktop\x86\release\bin\k4a.dll`
+		* `C:\Program Files\Azure Kinect SDK v1.2.0\sdk\windows-desktop\x86\release\bin\k4a.pyd`
+		* `C:\Program Files\Azure Kinect SDK v1.2.0\sdk\windows-desktop\x86\release\bin\depthengine_2_0.dll`
+* You have Python 3.6 or newer development libraries installed. 
+	* This may be installed on your computer simply as Python.  
+	* Note the following directories and add them to your user Path
+		* `C:\Users\amy\AppData\Local\Programs\Python\Python37\include`
+		* `C:\Users\amy\AppData\Local\Programs\Python\Python37\DLLs`
+		* `C:\Users\amy\AppData\Local\Programs\Python\Python37\libs`
+* You have a C compiler
+	* e.g., VS2019 with C++ desktop development tools, C++/CLI, and most recent Windows 10 SDK installed
+		* Make sure your environmental variables are set, i.e. CMAKE_C_COMPILER should point to cl.exe inside VS2019
+			* e.g., `C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\bin\Hostx86\x86\cl.exe`
+* Copy elements from the SDK to your Python directories and set environment and path variables
+	* Add following directory (equivalent on your machine) to your user Path
+		* `C:\Program Files\Azure Kinect SDK v1.2.0\sdk\include`
+		* at this time, make sure you also have the Python paths listed above
+	* Copy the k4a folder of the SDK to your Python "include" folder (see example directories above)
+	* Copy the k4a.dll and k4a.pyd of the SDK to your Python DLLs folder (see example directories above)
+	* Copy the k4a.lib to your Python lib folder (see example directories above)
+	* Create an environmental variable LINK pointing to k4a.lib, with quotes! 
+		* e.g., `"C:\Program Files\Azure Kinect SDK v1.2.0\sdk\windows-desktop\amd64\release\lib\k4a.lib"`

```

Comment:
Did you really use the .lib from the amd64? Above you use the x86 path

Reply:
I actually think I used both (one on each install), and it didn’t seem to matter. I definitely used the amd64 version on the first install (where I created this from) but think I switched to x86 on the second, since I also thought it was odd. Was just poking around in folders the first time around, so not as familiar!

Reply:
I actually think I used both (one on each install), and it didn’t seem to matter. I definitely used the amd64 version on the first install (where I created this from) but think I switched to x86 on the second, since I also thought it was odd. Was just poking around in folders the first time around, so not as familiar!

Reply:
Thanks, I finally got this working on windows. Turns out I was using the x86 files instead of amd64. 

I did not try to build this project but it should be the same. I am using the other k4a python wrapper: https://github.com/etiennedub/pyk4a

`pip install pyk4a --global-option=build_ext --global-option="-IC:\Program Files\Azure Kinect SDK v1.2.0\sdk\include" --global-option="-LC:\Program Files\Azure Kinect SDK v1.2.0\sdk\windows-desktop\amd64\release\lib"`

Reply:
Nice. I am still getting a few device errors with this one — but from looking at that example given on the other I think this module still has more of the SDK ported over. Would love to hear how the other one works for you!

This one = brendandburns
That one = etiennedub

Reply:
I did not try this one. I am only using the other one as I find it is simple code.
Using Linux I had no problem.
I just sent a pull request on the other one. It should work with pip on windows when it is merged and a new release comes out.